src/RL/utils/featuredgraph/featuredgraph.jl

```diff
@@ -0,0 +1,90 @@
+mutable struct FeaturedGraph{T <: AbstractMatrix, N <: AbstractMatrix, E <: AbstractArray, G <: AbstractVector} <: AbstractFeaturedGraph
+    graph::T
+    nf::N
+    ef::E
+    gf::G
+    directed::Bool
```

Comment:
What is the benefit of using parametric types `{T <: AbstractMatrix, N <: AbstractMatrix, E <: AbstractArray, G <: AbstractVector}` for the `FeaturedGraph` struct ? Is it from GeometricFlux ?

Reply:
Yes, it's from GeometricFlux.